app/controllers/people/person/personal-information/edit.js

```diff
@@ -29,7 +29,7 @@ export default class PeoplePersonPersonalInformationEditController extends Contr
 
     if (valid) {
       for (let contact of contacts) {
-        let { primaryContact, secondaryContact, address } = contact;
+        let { primaryContact, secondaryContact, address, _position } = contact;
```

Comment:
I am not sure why we should prefix `_position` here ? I tried to look for some doc about prefixing with an underscore but I didn't find :sweat_smile:

Reply:
I ddidn't have a really good reason 😄 .
I wanted  to say "this is the underlying ember data position that can be either a minister or mandatory" 
Do you think it makes sense?